ios/Plugin/Plugin.swift

```diff
@@ -28,6 +28,9 @@ public class GoogleAuth: CAPPlugin {
         if let forceAuthCodeConfig = getConfigValue("forceCodeForRefreshToken") as? Bool {
             forceAuthCode = forceAuthCodeConfig;
         }
+        if let forceAuthCodeConfig = getConfigValue("forceCodeForRefreshTokenIOS") as? Bool {
```

Comment:
Is it valid in swift to declare a variable twice like this?

Reply:
I think the declared variable is scoped to the `if` block